File: mini-package-gui/src/app/models/offer.ts
export class Offer {
  offerId: number;
  price: number;
  teamName: string;
  offerStatus: string;

  constructor() {
    this.offerId = 0;
    this.price = 0;
    this.teamName = "";
    this.offerStatus = "";
  }
}

File: mini-package-api/mini-package-api/Models/Offer.cs
using mini_package_api.Enums;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace mini_package_api.Models
{
    public class Offer
    {
        public int OfferId { get; set; }

        public int InquireId { get; set; }

        public virtual Inquire Inquire { get; set; }

        public decimal Price { get; set; }

        public TeamNameEnum TeamName { get; set; }

        public string OfferStatus { get; set; }

        public int CourierId { get; set; }

        public virtual Employee Courier { get; set; }

        public Guid Guid { get; set; }
    }
}
